app/controllers/editions_controller.rb

```diff
@@ -153,4 +169,11 @@ def progress_action_param
   def permitted_params
     params.require(:edition).permit(%i[title overview in_beta body major_change change_note])
   end
+
+  def destroyable_edition?
+    return if @resource.can_destroy?
+
+    flash[:danger] = "Cannot delete a #{description(@resource).downcase} that has ever been published."
```

Comment:
Is there any way that we can re-word this? Just worried that we could display something grammatically incorrect such as:
"Cannot delete a answer that has ever been published"

Reply:
This has been copied from existing error message and I think we have decided not to change error messages for now.

I have made the note of this to be added to the doc where we are maintaining list of all error messages to be reviewed.